src/features/common/Layout/CookiePolicy/index.tsx

```diff
@@ -31,7 +32,11 @@ export default function CookiePolicy() {
     localStorage.setItem('cookieNotice', showCookieNotice);
   }, [showCookieNotice]);
 
-  return ready && showCookieNotice ? (
+  const router = useRouter();
+  const isEmbed = router.query.embed === 'true';
+
+
+  return ready && showCookieNotice && !isEmbed? (
```

Comment:
@prachigarg19 It may be better to conditionally render `CookiePolicy` and `RedeemPopup` within `/src/features/common/Layout/index.tsx` instead of adding this logic within both `CookiePolicy` and `RedeemPopup` as it keeps the code cleaner and avoids duplication.